kitchen.dokken.yml

```diff
@@ -0,0 +1,23 @@
+---
+driver:
+  name: dokken
+  privileged: true  # because Docker and SystemD/Upstart
+  chef_image: cincproject/cinc
+  chef_version: <%= ENV['CHEF_VERSION'] || '17' %>
```

Comment:
```suggestion
  chef_version: <%= ENV['CHEF_VERSION'] || '18' %>
```

Reply:
I've updated the dokken template, so this should no longer happen in the future.